student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parents_permission: true)
```

Comment:
- [ ] Similar to the default value in the name please 🙏🏽 use "=" to define the default value to `parent_permission` for the correct syntax.